src/app/chat/chat-list/chat-list.component.ts

```diff
@@ -82,43 +99,72 @@ export class ChatListComponent extends RouterEnter {
     }
   }
 
+  // force every navigation happen under radar of angular
+  private _navigate(direction) {
+    if (this.utils.isMobile()) {
+      // remove name from passing values
+      // this name is receiver name, we pass this to set to header in char room.
```

Comment:
yes, after we change `_getChatName()` method logic to what you mention in another comment we no need to remove it. it will handle from `_getChatName()`.